README.md

```diff
@@ -1,2 +1,43 @@
-# KOBAW
-KOB Association Wallet
+
+| Title | KOB Association Wallet |
+|--- | ---|
+| Version | TBD |
+
+
+# Abstract   
+
+The KOB Association Wallet is an association/organisational wallet for the city of Kochi. This holds the verifiable credentials about registered entities (organisations) under a particular jurisdiction. And this is a Web Application hosted on the Community Cloud for the City of Kochi. A wallet provides an interface for storing and accessing identity information.
```

Comment:
And this is a Web Application hosted on the Community Cloud for the City of Kochi - This is incorrect. Please take it out.